sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilters.scala

```diff
@@ -126,264 +126,270 @@ class ParquetFilters(
     Binary.fromConstantByteArray(fixedLengthBytes, 0, numBytes)
   }
 
-  private val makeEq: PartialFunction[ParquetSchemaType, (String, Any) => FilterPredicate] = {
+  private val makeEq
+  : PartialFunction[ParquetSchemaType, (Array[String], Any) => FilterPredicate] = {
```

Comment:
nit. indentation.

Reply:
Adding two space here?

Should I move `:` to previous line?

```scala
  private val makeEq:
    PartialFunction[ParquetSchemaType, (Array[String], Any) => FilterPredicate] = {
```

Reply:
Why don't we have a `type` like in the previous PR?

Reply:
`Adding two space here?` is correct.

Reply:
I decide not to do type alias because it's easier to see the actual type definition of function.